src/types/file.ts

```diff
@@ -8,10 +8,10 @@ export const types = {
 	},
 };
 
-type WriteType = Buffer | string;
+export type Types = TypeUtils.TsTypes<Buffer, Buffer | string>;
```

Comment:
So we're now forcing a ReadType be defined for each sbvr type? I've seen a number of cases where they didn't have one defined before but now do because `TypeUtils.TsTypes` requires it.

Reply:
Yes, basically it's the type we'll return when fetching from the database after any fetch processing that might happen. Which is equivalent to the type that will be available on the OData response (assuming no hooks or anything sit in the middle and modify it)